e2e-app/src/app/datepicker/datepicker.po.ts

```diff
@@ -7,9 +7,9 @@ export class DatepickerPage {
 
   getToggle() { return $('#toggle'); }
 
-  getDayElement(date: Date) {
+  getDayElement(date: Date, index = 0) {
```

Comment:
This change is very confusing from the readability point of view to me.

Could we replace index with the datepicker, so the test would look something like this (instead of using indexes):

```
const datepicker1 = page.getDatepicker('myDatepicker1');
const element = page.getDayElement(date, datepicker1);
```